### ISSUE TITLE
Really lacking in info

### ISSUE THREAD

**Issue by Slaverty6** [MicrosoftDocs/OfficeDocs-DeployOffice#716]
This page leaves a lot up to guess work, resulting in frequent googling error messages to find out what steps have been excluded from these instructions. Such as confusion over the contents of the XML, whether a seperate XML is required per download/config or if one will do, how to check when download finished, where download downloads to, where download should be for install ect ect. Really, this isn't even half effort.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6da18a8b-9e96-0155-7b30-9c211070d793
* Version Independent ID: 5a7e610a-bfcb-d9cc-03f7-589d970128d9
* Content: [Deploy Office 365 ProPlus by using Remote Desktop Services](https://docs.microsoft.com/en-us/deployoffice/deploy-office-365-proplus-by-using-remote-desktop-services#feedback)
* Content Source: [DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/deploy-office-365-proplus-by-using-remote-desktop-services.md)
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by RAJU2529** [comment]
@Slaverty6 . @DHB-MSFT is clearly mentioned the site link, to configure and download the office suite using office deployment tool.
**https://docs.microsoft.com/en-us/deployoffice/overview-of-the-office-2016-deployment-tool**

The above link is also mentioned in between paragraph in this article.

**Comment by Slaverty6** [comment]
Yeah, no sh**, I know how a link works, I was also referring to that page in my comment.  It's a joke, anyone new to this is going to have a very hard time trying to figure it out.  I mean, on that page you linked to, the official method of determining if your download has completed successfully is to "go to the download location you defined in the configuration file and look for an Office folder with the appropriate files in it.".    Oh yeah, that's real helpful.  Just look for the files.  Even though you don't know what files should be there, or what sizes they should be, yeah, just look and if you see them then you're good to go.  It's a joke!

**Comment by RAJU2529** [comment]
@Slaverty6 Last weak i downloaded office 2019 suite by configuring .xml file, and downloaded successfully,
also I configured  .xml to another user to download office 365 suite, so I forwarded to him, he also successfully and installed fully. The problem is solved.

here is the configuration with setup file 
[Office365.zip](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/files/4414296/Office365.zip)
